realm/realm-library/src/objectServer/java/io/realm/mongodb/Credentials.java

```diff
@@ -150,17 +152,33 @@ public static Credentials facebook(String accessToken) {
     }
 
     /**
-     * Creates credentials representing a login using a Google Authorization Code.
+     * Creates credentials representing a login using a Google access token of a given {@link GoogleAuthType}.
+     * <p>
+     * This provider must be enabled on MongoDB Realm to work.
+     *
+     * @param accessToken the access token returned when logging in to Google.
```

Comment:
We should probably just rename this to `token` as the type is determined by the enum?